server/modules/module_state_machine.js

```diff
@@ -53,7 +53,16 @@ class ModuleStateMachine extends stateMachine.Machine {
       clients: Object.keys(allClients)
           .map(k => allClients[k])
           .filter(c => isDisplayInPoly(c.getClientInfo().rect, geo)),
-    })
+    });
+    
+    // Forward errors from my child state machines to my listener.
+    this.context_.server.setErrorListener(error => this.errorListener_(error));
+    this.context_.clients.forEach(c => c.setErrorListener(error => this.errorListener_(error)));
+    this.setErrorListener(error => {
+      debug('Error in sub-sm, continuing playlist...');
```

Comment:
You never log the error?  how about debug(error)?

Reply:
We log the error in the catch block in lib/state_machine.js, so we don't re-log it here.